src/backend/web/renderers/team_renderer.py

```diff
@@ -321,6 +334,12 @@ def render_team_details(
             "max_year": SeasonHelper.get_max_year(),
             "hof": hall_of_fame,
             "team_district_points": team_district_points,
+            "has_any_pit_location": any(
+                comp["event"].official
```

Comment:
Do we care about the official filter? I think some offseasons run nexus. (also I think `.official` has lost its real meaning, i think now it just means FMS sync is enabled)

Reply:
I think this is something we can relax in the offseason (and maybe ask @eforbes for an API to fetch all offseason events), so we don't have to optimistically try to hit every offseason looking for data.